mmdet/models/data_preprocessors/data_preprocessor.py

```diff
@@ -12,11 +12,14 @@
 from mmengine.model import BaseDataPreprocessor, ImgDataPreprocessor
 from mmengine.structures import PixelData
 from mmengine.utils import is_list_of
+from skimage import color
```

Comment:
skimage is not the default requirement package of mmdet. You should import it like this:

```
try:
    import skimage
except ImportError:
    skimage = None
```

and check it in the __init__ of BoxInstDataPreprocessor like this:
```
    def __init__(self, ...):
        if skimage is None:
            raise RuntimeError(
                'skimage is not installed, please install it by: ...'
```

Reply:
Maybe we can use cv2?